cas/cas.go

```diff
@@ -238,14 +245,48 @@ func (ds Store) WriteACI(r io.Reader) (string, error) {
 			BlobKey:    key,
 			AppName:    im.Name.String(),
 			ImportTime: time.Now(),
+			Latest:     latest,
 		}
 		return WriteACIInfo(tx, aciinfo)
 	}); err != nil {
 		return "", fmt.Errorf("error writing ACI Info: %v", err)
 	}
+
+	// The treestore uses acirenderer that needs cas.ResolveKey that will
+	// need the aciinfo already written in the db.
```

Comment:
Yeah, I understand the lines below my comment. The part I don't get is "The treestore uses acirenderer that needs cas.ResolveKey that will need the aciinfo already written in the db.".

Are you trying to make explicit why you need to write aciinfo?